docs/advanced/features/external-logins/index.mdx

```diff
@@ -67,14 +67,14 @@ module.exports = {
 ## Magento1 Requirements
 
 For the external logins to work with the Magento1 module in addition to
-[Adding the necessary modules](#Adding-the-necessary-modules) you need to ensure
-you are have the magento1 connector on 1.4.2 or later.
+[Adding the necessary modules](#adding-the-necessary-modules) you need to ensure
```

Comment:
actually anchors must now be in lowercase and we have a lot in the documentation… I'll open a different MR to fix others

Reply:
same fix done in the rest of the documentation in https://github.com/front-commerce/developers.front-commerce.com/pull/465